### PR TITLE
Rs 2022 08 14

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 1]
```diff
@@ -73,17 +73,17 @@ The pain in my eyes is indescribable.
 -- 眼孔に火の玉が入っているかのようだ。
 // >says it cant be described
 // >describes it
-As if balls of fire were inserted into my eye sockets.
+It's as if fireballs have been inserted into my eye sockets.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4B.txt:76]: I like "balls of fire" here

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 2]
```diff
@@ -189,17 +191,20 @@ That's why I'm risking my life to help her.
...
-Me, running on the ground, or the Executor covering me from above?
+Will he go for me, running along the ground?%{n}
+Or for the Executor, covering me from above?
 }
 [sha:b6a8c9e9473e49699db237566efc12db639d4e19]{
 -- Page 743, Offset 12875.
 -- 瞬時の判断で、ヤツは先輩こそ危険だと理解した。自分を殺すのはあの女だと。
-In a split-second judgment, he realizes that Senpai is the true hazard here. That she will be the one to kill him.
+// Keeping scale metaphor alive a bit
+Weighing the options for a split second, he grasps that Senpai is the real danger here.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4B.txt:206]: After weighing the options for a split second,

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 3]
```diff
@@ -209,27 +214,35 @@ However―――
 [sha:73176aa278552bbeda59821351045e7e8c94a89e]{
 -- Page 744, Offset 12877.
 -- だが、そのすべての直感を排斥して、ヤツは地を這う俺を選んだ。
-However, rejecting all of his intuition, he goes for me who is creeping on the ground.
+// Another instance of just _awkward_ state tagging in EN with 地を這う.
+// Really needs to be broken out into another thought or just dropped.
+However, in a complete rejection of his intuition, he chooses me instead.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4B.txt:219]: prefer introducing em dash but colon could work too
me instead--the one creeping across the ground toward him.
me: the one creeping across the ground toward him.

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 4]
```diff
@@ -239,22 +252,22 @@ That's why you came, all alone, to such a foreign country.
 [sha:88dae8a80b32fab25e6c28ca907590ecd93525a5]{
 -- Page 747, Offset 12883.
 -- 「“枷は解け、骨は外れ、仇は成せり”―――」
-"Shackles are broken, bones are removed, grudges are borne―――"
+"Shackles are released, bones are dislocated, grudges are borne―――"
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4B.txt:255]: maybe semicolons

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 5]
```diff
@@ -264,48 +277,50 @@ As if to show me the way―――
 [sha:ea258243fef18a7fb2ac628f3120d44d17cbca18]{
 -- Page 749, Offset 12888.
 -- 「“その遠吠えは　万里を手繰る―――！”」
+// There lore context to inform this, or just chuuni speak?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4B.txt:280]: ask lore council. I can't find anything relevant obvious in the script

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 6]
```diff
@@ -336,33 +351,33 @@ The pain, originating from the 'point' I stabbed, is transmitted through the kni
...
 [sha:084e581666228a45521cbc76945a014d575e392b]{
 -- Page 753, Offset 12905.
 -- 失敗した―――あの輝きではなくヤツの死だけを視ていれば、例外なく“殺し”きれたものを―――！
-I failed―――If I saw his death instead of that radiance, I would have definitely killed him―――!
+I failed. If I'd been looking at his death instead of at that radiance, I'd have 'killed' him for sure―――！
 }
 [sha:f21f4f6d315477dd1d34789ea99858654cc61937]{
 -- Page 753, Offset 12906.
 -- 「“―――何者だ―――！”」
 // reads like hes trying to say smth like "What are you trying to do" without italics imo
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4B.txt:374]: delete comment

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 7]
```diff
@@ -382,17 +397,17 @@ The serpentine sword mows them down.
...
 -- Page 757, Offset 12912.
 -- 音をたてて回転する断罪刃。
-The beheading sword whooshes as it rotates.
+The sword of judgment makes a whirring noise as it spins.
 }
 [sha:f9f88b3119d2b94d368005679c52cf5a5a4c81ac]{
 -- Page 757, Offset 12913.
 -- 代行者を中心にして発生した竜巻は、吸血鬼の体を上空に打ち上げ、そして―――
-The tornado that forms with the Executor at its center lifts the body of the vampire into the sky, and―――
+It forms a tornado, with the Executor at its center, that launches the body of the vampire into the sky, and then―――
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4B.txt:410]: drop first comma i reckon

**Review by Avuxo — APPROVED** [review]

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 8]
```diff
@@ -209,27 +214,35 @@ However―――
...
-Anger and the like are too feeble. It is his obsession, for which he risks his entire existence, his pride that made him choose to kill me.
+Petty emotions such as anger cannot command this intensity.
+ It is his pride, it is the obsession on which he pins his entire existence, that drove him to choose to kill me.
 }
 [sha:bd3deccd6f4c60d72210d8674f3b4c80b22cf3aa]{
 -- Page 746, Offset 12881.
 -- ああ―――素晴らしいが、なんて愚か。
-Ahhh―――How beautiful, but how foolish.
+%{i}Ahhh%{/i}―――%{n}
+How beautiful, but how foolish.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_4B.txt:245]: This is internal Shiki dialog

**Pull request merged**